app/user/rest.py

```diff
@@ -491,6 +498,10 @@ def send_branding_request(user_id):
         current_app.logger.error(e)
         return jsonify({}), 400
 
+    status_code = Zendesk(contact).send_ticket()
+    if status_code != 201:
+        current_app.logger.error(f"Zendesk error response code: {status_code}")
```

Comment:
The `send_ticket` function does not return status code and that would not be the proper way to catch the error. We'd rather need to catch the exception and log along with the status code.

Reply:
Nice find, i totally missed this when i refactored error handling. Will add the try/catch

Reply:
That's a lot of changes we went through so quite understandable. Our many 👀 are here to help. 👍